.github/workflows/docker_build.yml

```
@@ -1,458 +1,458 @@
-          echo "=== Nginx logs ==="
-          docker compose -f docker-compose.prod.yml -f docker-compose.ci.yml logs nginx --tail=30
-
-      - name: Tear down
-        if: always()
-        run: |
-          docker compose \
-            -f docker-compose.prod.yml \
-            -f docker-compose.ci.yml \
-            down -v --remove-orphans
\ No newline at end of file
+#name: CI
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)